### PR TITLE
[ADD] Order of Events in chatter

### PR DESCRIPTION
**Pull request by eilst**

### CODE REVIEW

File: project_event/models/project_project.py [hunk 1]
```diff
@@ -240,3 +240,27 @@ def map_tasks(self, new_project_id):
                 'project_id': new_project_id}
             tasks += task.copy(defaults)
         return self.browse(new_project_id)
+    
+    @api.multi
+    def _message_track(self, tracked_fields, initial):
+        mail_track = super()._message_track(tracked_fields, initial)
+        import ipdb; ipdb.set_trace()
```
Comment (eilst) [inline project_event/models/project_project.py]:
```suggestion
```

File: project_event/models/project_project.py [hunk 2]
```diff
@@ -240,3 +240,26 @@ def map_tasks(self, new_project_id):
...
+    def _message_track(self, tracked_fields, initial):
+        mail_track = super()._message_track(tracked_fields, initial)
+        changes = mail_track[0]
+        tracking_value_ids = mail_track[1]
+        order_fields = self.order_event_fields(tracking_value_ids)
+        return changes, order_fields
+
+    @staticmethod
+    def order_event_fields(tracking_values):
+        event_fields_list = [
```
Comment (rimbendhaou) [inline project_event/models/project_project.py:254]: Field order: 'state',  'name', 'code', 'responsible_id', 'partner_id', 'notes'

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/models/project_project.py (hunk 2 above)
Comment (eilst) [reply project_event/models/project_project.py:254]: :+1:

File: project_event/models/project_project.py [hunk 3]
```diff
@@ -240,3 +240,30 @@ def map_tasks(self, new_project_id):
...
+            'state',
+            'name',
+            'code',
+            'responsible_id',
+            'partner_id',
+            'notes'
```
Comment (eilst) [inline project_event/models/project_project.py]:
```suggestion
            'user_id',
            'notes'
```

Comment (eilst) [inline project_event/models/project_project.py]:
```suggestion
            'notes',
            'user_id'
```

File: project_event/models/project_project.py (hunk 2 above)
Comment (eilst) [reply project_event/models/project_project.py:254]: though, user_id must also be here

Comment (rimbendhaou) [reply project_event/models/project_project.py:254]: chatter doesn't record any new modifications (only track the fields of the new created event)

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/models/project_project.py (hunk 2 above)
Comment (eilst) [reply project_event/models/project_project.py:254]: I check this and work well in my local, and the tests pass, @lgharib does it work in you machine?
Thanks

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**